boards/remote-pa/doc.txt

```diff
@@ -1,5 +1,89 @@
 /**
-@defgroup    boards_remote-pa Re-Mote Prototype A
+@defgroup    boards_remote-pa RE-Mote Prototype A
```

Comment:
Cannot the `README.md` be turned into a doc.txt and use it as documentation?

Reply:
They differ unfortunately.

Reply:
The revision of the documentation for RE-Mote nodes is outside the scope of this PR.

Reply:
Agreed.